### DISCUSSION TITLE
Switch to large disk version

### DISCUSSION THREAD

**Discussion by uncarman2017** [seaweedfs/seaweedfs#2897] [Q&A]
**Describe the bug**
Can the data files of the low capacity version (30GB) of seaweedfs not be hard copied to the data directory of the large disk version (8000GB)?

**Comment by uncarman2017** [comment]
Previously, I used seaweedfs 2.91 (30g) to copy the data file to v2 After 95 (8000g) data directory, it is abnormal. What is the reason and how to solve it

**Comment by chrislusf** [comment]
the `.dat` file format is the same, but `.idx` files are not. Need to run `weed fix` for each volume manually if you must move from normal version to large_disk version.

**Reply by uncarman2017** [reply]
weed fix -volumeId=1 /tmp
is the command above correct? and should I run it in the data folder of each volume server?  
I try the "weed" of large disk version just now, but  it's not working.

**Reply by chrislusf** [reply]
Correct if your files are under /tmp.

**Reply by uncarman2017** [reply]
It's working！thks.

**Reply by Gnaiqll** [reply]
> the `.dat` file format is the same, but `.idx` files are not. Need to run `weed fix` for each volume manually if you must move from normal version to large_disk version.

请问下换成最新的3.96版本还需要给每个volume运行weed fix吗

**Reply by chrislusf** [reply]
correct
